docs/excel/excel-add-ins-troubleshooting.md

```diff
@@ -41,6 +41,56 @@ The following APIs are affected by this workbook switch:
 
 See [Coauthoring in Excel add-ins](co-authoring-in-excel-add-ins.md) for patterns to use with events in a coauthoring environment. The article also discusses potential merge conflicts when using certain APIs, such as [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-).
 
+## Known Issues
+
+### Binding events ID discrepancy
```

Comment:
I'm not sure "discrepancy" is how we should describe this. It makes it sound like a bug instead of an async design limitation. We could make the header reflect the underlying issue with something like "Binding events return temporary binding objects" or "Getting bindings from the temporary binding objects passed to events".